backend/src/server.js

```diff
@@ -1,57 +1,22 @@
 require('dotenv').config({ path: '.env' });
-const userRoutes = require('./routes/user');
-const { validateToken } = require('./helpers/auth');
+const serverConfig = require("./helpers/config");
 const port = process.env.SERVER_PORT || 8080;
-const jwksUri = process.env.JWKSURI;
-const fastifyCors = require('fastify-cors');
-const fastifyAuth = require('fastify-auth');
-const fastify = require('fastify')({
+
+// Save server configuration and enable logging.
+const server = serverConfig(({
     logger: {
         level: 'info',
         // We can output logs to a file with fastify's default logger.
         // file: '/path/to/file'
     }
-});
-
-// Fastify server config.
-fastify
-  .decorate('verifyJWT', (req, res, done) => {
-      validateToken(jwksUri, req)
-        .then(() => done())
-        .catch((err) => done(new Error(err)));
-  })
-  .register(fastifyAuth)
-  .register(fastifyCors, {})
-  .register(userRoutes)
-  .after(() => {
-    fastify.addHook('preHandler', fastify.auth([
-        fastify.verifyJWT,
-    ]));
-
-    // Register root route.
-    fastify.route({
-        method: 'GET',
-        url: '/',
-        handler: async () => {
-            return { hello: 'world' }
-        }
-    });
-
-    fastify.route({
-        method: 'GET',
-        url: '/api/health',
-        handler: async () => {
-          return { health: 'good' }
-        }
-    });
-  })
+}));
 
```

Comment:
in my upcoming PR ive got some boilerplate in here to handle signals from the OS in order to gracefully shut down, stand up the db connection and check the db, etc. you and i should touch base about where some of that stuff should go, now that you have made this helper thing, because i was uncomfortable about how big/messy it was getting in here.